src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,104 @@
+/*  eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import classnames from 'classnames'
+
+import { Footer } from './Footer'
+
+export default {
+  title: 'Footer',
+  parameters: {
+    info: `
+  USWDS 2.0 Footer component
+  
+  Source: https://designsystem.digital.gov/components/form-controls/#footer
+  `,
+  },
+}
+
+type SizeProps = {
+  big?: boolean
+  medium?: boolean
+  slim?: boolean
+}
+
+const Address = (): React.ReactElement => (
+  <address className="usa-footer__address">
+    <div className="grid-row grid-gap">
+      <div className="grid-col-auto mobile-lg:grid-col-12 desktop:grid-col-auto">
+        <div className="usa-footer__contact-info">
+          <a href="tel:1-800-555-5555"> (800) CALL-GOVT</a>
+        </div>
+      </div>
+      <div className="grid-col-auto mobile-lg:grid-col-12 desktop:grid-col-auto">
+        <div className="usa-footer__contact-info">
+          <a href="mailto:info@agency.gov"> </a>info@agency.gov
+        </div>
+      </div>
+    </div>
+  </address>
+)
+
+// Placeholder until a dynamic Nav is built out and can be used
+const MockNav = ({ medium, slim }: SizeProps): React.ReactElement => {
+  const itemClasses = classnames(
+    'desktop:grid-col-auto usa-footer__primary-content',
+    {
+      'mobile-lg:grid-col-4': medium,
+      'mobile-lg:grid-col-6': slim,
+    }
+  )
+
+  const items = Array(4).fill({
+    href: 'javascript:void(0);',
+    heading: 'Primary Link',
+  })
+
+  return (
+    <nav className="usa-footer__nav" aria-label="Footer navigation">
+      <ul className="grid-row grid-gap">
+        {items.map((item, i) => (
+          <li key={`navItem-${i}`} className={itemClasses}>
+            <a className="usa-footer__primary-link" href={item.href}>
+              {item.heading}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </nav>
+  )
+}
+
+const Logo = ({ big, medium, slim }: SizeProps): React.ReactElement => {
+  const containerClasses = classnames('usa-footer__logo grid-row', {
+    'mobile-lg:grid-col-6 mobile-lg:grid-gap-2': big || medium,
+    'grid-gap-2': slim,
+  })
+
+  return (
+    <div className={containerClasses}>
+      <div className="grid-col-auto">
+        <img className="usa-footer__logo-img" src="#" alt="img alt text" />
```

Comment:
🤔 Should we add a mock logo image file to our library? It might be nicer for use in stories.

Reply:
Yeah! we could use the empty circle used on USWDS for something easy? https://designsystem.digital.gov/components/footer/

as long as it's only imported to the `.stories.tsx` file it won't be included in the packaged library.

Reply:
Just pushed a couple more commits to round out this PR. Ended up adjusting the `npm run storybook` script to get the static logo image loading properly.  Followed [this ](https://storybook.js.org/docs/configurations/serving-static-files/#2-via-a-directory)approach.